library/std/src/sys/unix/process/process_common.rs

```diff
@@ -128,6 +169,47 @@ pub enum Stdio {
     Fd(FileDesc),
 }
 
+pub struct SignalSet {
```

Comment:
Shouldn't this be private?

Reply:
It follows the pattern of the other items in the module which are all declared `pub` (though they actually resolve to `pub(crate)`).